control-plane/config/crd/bases/consul.hashicorp.com_httproutetimeoutfilters.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
sorry, think this file needs to be deleted too, looks like it's part of the old CRD before you renamed it.